mods/parsoid.js

```diff
@@ -715,6 +715,22 @@ PSP.makeTransform = function(from, to) {
                     status: e.status
                 });
             }
+            // In case the a page was deleted/revision restricted wkile edit was happening,
```

Comment:
Hmm, while it is true that the code above will disappear soon, currently it will log something other than 429 (either a 404 or a 403). Should we try to integrate these two so that we can easily match user reports with logs (which was the initial incentive to introduce this temp code) ?

Also, some small nits on this line:
`s/the//`
`s/wkile/while/`